buildSrc/src/main/java/org/elasticsearch/test/NamingConventionsCheck.java

```diff
@@ -69,6 +69,10 @@ public static void main(String[] args) throws IOException {
                     fail("unsupported argument '" + arg + "'");
             }
         }
+        if (rootPathList == null) {
+            fail("No paths provided");
+            return;
```

Comment:
There should be no need for a return here since fail throws an exception.

Reply:
It calls `System.exit` which is not picked up and IDE complains below that rootPathList might be null.